api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,232 @@
+import logging
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import AnalyzerRunException, FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class CAPEsandbox(FileAnalyzer):
+    def set_params(self, params):
+        self.__token = self._secrets["api_key_name"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+        self.max_tries = params.get("max_tries", 50)
+        self.poll_distance = params.get("poll_distance", 30)
+        self.__base_url = self._secrets.get(
+            "url_key_name", "https://www.capesandbox.com"
+        )
+        self.__session = requests.Session()
+        self.__session.headers = {
+            "Authorization": f"Token {self.__token}",
+        }
+
+    def run(self):
+        api_url: str = self.__base_url + "/apiv2/tasks/create/file/"
+        to_respond = {}
+
+        try:
+            response = self.__session.post(
+                api_url,
+                files={
+                    "file": (self.filename, self.read_file_bytes()),
+                },
+                data={"machine": self.__vm_name},
+            )
+            response.raise_for_status()
+        except requests.RequestException as e:
+            raise AnalyzerRunException(e)
+
+        logger.info(f"[INFO] (Job: {self.job_id} -> Uploaded the file to CAPESandbox")
```

Comment:
there is no need to add the level of the log in the message. This is already done by the formatter.